test_ai_aggies.py

```diff
@@ -15,6 +15,25 @@ def test_anonymize_nrp(self):
 
         self.assertEqual('My NRP is Unknown',
                          anonymize_text('My NRP is Unknown', ['NRP']))
+6-detecting-location-though-gps-and-indirect-sources
```

Comment:
It looks like you tried to resolve merge conflicts and introduced this line by mistake.

Reply:
ok I removed it